packages/paste-website/src/components/icons-list/IconsList.tsx

```diff
@@ -189,9 +185,14 @@ const IconsList: React.FC<IconsListProps> = () => {
             </Card>
           )}
         </Column>
-        <IconColumn paddingX="space60" position="sticky" top={STICKY_COLUMN_OFFSET}>
+        <Box
+          paddingX="space60"
+          position="sticky"
```

Comment:
It looks like this position sticky isn't working anymore, but its not working on the production site either: https://paste.twilio.design/icons

We can create another ticket to figure out that issue.

Reply:
Yeah, that confused me too. I can't remember if we removed it or it broke at some point.